cli/tests/test_config_command.rs

```diff
@@ -621,12 +623,12 @@ fn test_config_edit_user() {
     let repo_path = test_env.env_root().join("repo");
     let edit_script = test_env.set_up_fake_editor();
 
-    std::fs::write(
-        edit_script,
-        format!("expectpath\n{}", test_env.config_path().to_str().unwrap()),
-    )
-    .unwrap();
+    std::fs::write(edit_script, "dump-path path").unwrap();
     test_env.jj_cmd_ok(&repo_path, &["config", "edit", "--user"]);
+
+    let edited_path = std::fs::read_to_string(test_env.env_root().join("path")).unwrap();
+    let edited_path = Path::new(&edited_path);
```

Comment:
optional (here and below): use `PathBuf` here instead and avoid the intermediate variable (sorry, I should have thought of this when i suggested defining `edited_path` as a `Path`)

Reply:
Done.